bindings/nodejs/lib/gpio.js

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
@barbieri The problem is that npm packages do not have an installation step. The repo *is* the installed package. ```npm install``` merely modifies the repo in place to prepare it for use. We could copy the files to the root of the repo as part of ```npm install```. I guess we can keep the file where it is for now, and I'll experiment with locations when I do the OIC JS API.